### PR TITLE
docs: add casing support documentation 🍆

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates to keymanapp/keyman#3620 and keymanapp/keyman#3621.

### CODE REVIEW

File: cdn/dev/js/prism-keyman.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ Prism.languages.keyman = {
 	'function': /\[\s*((CTRL|SHIFT|ALT|LCTRL|RCTRL|LALT|RALT|CAPS|NCAPS)\s+)*([TKU]_[a-z0-9_?]+|".+?"|'.+?')\s*\]/i,  // virtual key
 	'string': /("|')((?!\1).)*\1/,
 	'keyword': /\b(any|baselayout|beep|call|context|deadkey|dk|if|index|layer|notany|nul|outs|platform|return|reset|save|set|store|use)\b/i,  // rule keywords
-	'atrule': /\b(ansi|begin|unicode|group|using keys|match|nomatch)\b/i,   // structural keywords
+	'atrule': /\b(ansi|begin|unicode|postkeystroke|newcontext|group|readonly|using keys|match|nomatch)\b/i,   // structural keywords
```
Comment (darcywong00) [inline cdn/dev/js/prism-keyman.js:6]: TODO: PR to https://github.com/PrismJS/prism/blob/master/components/prism-keyman.js

File: developer/language/guide/casing-support.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+---
```
Comment (darcywong00) [inline developer/language/guide/casing-support.md:1]: Should we add links for these new files to /developer/language/guide/index.php and /developer/language/reference/index.md?

Comment (mcdurdin) [reply developer/language/guide/casing-support.md:1]: Good catch! (I might rewrite index.php as index.md at the same time)

File: developer/language/reference/layerchanged.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    if(&layerChanged = "0") > use(nextLayer)
+
+group(nextLayer) readonly
+    num > layer('shift')
+    '. ' > layer('shift')
+```
+
+## Platforms
+
+While the `&LayerChanged` store is accessible on all platforms, in only has
```
Comment (darcywong00) [inline developer/language/reference/layerchanged.md:56]:
```suggestion
While the `&LayerChanged` store is accessible on all platforms, it only has
```

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**